### PR TITLE
created features

### PR DESCRIPTION
**Pull request by Joicyg**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+	def new
+	end
+	def create
+
```
Comment (eshaiju) [inline app/controllers/posts_controller.rb:12]: Please align

**Review by eshaiju — COMMENTED** [review]
Please use meaningful commit messages.

**Comment by Joicyg** [comment]
Ok will do that...

**Comment by Joicyg** [comment]
Hi Shaiju,
Please find the below updated Github link of last day assignment. And let
me know if some updations are yet to be done.


Link : https://github.com/Joicyg/BlogAppz/commits/create_feature

Thankyou.

**Pull request merged**